### PR TITLE
Update Part-I.md

### PR DESCRIPTION
**Pull request by xyfy**
Add missing steps

### CODE REVIEW

**Comment by hikalkan** [comment]
This step is not necessary when you configure your db mapping via fluent API as done in the tutorial. If you you data annotation attributes, then yes you need it. Right?

**Comment by xyfy** [comment]
do you mean this code below?
```
builder.Entity<Book>(b =>
            {
                b.ToTable(PersonPasswordHoldConsts.DbTablePrefix + "Books", PersonPasswordHoldConsts.DbSchema);
                b.ConfigureByConvention(); //auto configure for the base class props
                b.Property(x => x.Name).IsRequired().HasMaxLength(128);
            });
```

I try this ,but when I do not change the `XXXMigrationsDbContext` in `.EntityFrameworkCore.DbMigrations`, 
`Add-Migrations` take no effect, 

the `Down` and `Up` method in new migration class inside the Migrations folder  will be empty.

File: docs/en/Tutorials/AspNetCore-Mvc/Part-I.md [hunk 1]
```diff
@@ -107,7 +107,19 @@ builder.Entity<Book>(b =>
 
 #### Add New Migration & Update the Database
 
-The Startup template uses [EF Core Code First Migrations](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/) to create and maintain the database schema. Open the **Package Manager Console (PMC)** (under the *Tools/Nuget Package Manager* menu), select the `Acme.BookStore.EntityFrameworkCore.DbMigrations` as the **default project** and execute the following command:
+The Startup template uses [EF Core Code First Migrations](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/) to create and maintain the database schema. 
+
+You should also add a `DbSet` property to the `BookStoreMigrationsDbContext` class in the `Acme.BookStore.EntityFrameworkCore.DbMigrations` project, as shown below:
+
+````C#
+    public class PersonPasswordHoldMigrationsDbContext : AbpDbContext<PersonPasswordHoldMigrationsDbContext>
```
[the remark this answers is not in the document]
Comment (xyfy) [reply docs/en/Tutorials/AspNetCore-Mvc/Part-I.md]: (⊙﹏⊙)   It's mine project name, I will change it

**Comment by xyfy** [comment]
I am sorry ,but I take the fluentApi to error class.

**Pull request closed without merging**